packages/x-data-grid/src/models/gridBaseSlots.ts

```diff
@@ -319,3 +325,9 @@ export type TooltipProps = {
   enterDelay?: number;
   title: React.ReactNode;
 };
+
+export type ChipProps = {
+  label: string;
+  size?: 'small' | 'medium';
+  variant?: 'outlined' | 'filled';
+};
```

Comment:
I only see `variant="outlined"` being used, is `variant` useful to have in the base typings? In the rest of the base components, we have removed `variant` as much as possible.

Reply:
`variant="filled"` is the default for the Material UI component, so we need to set it to `outlined` somewhere. Would it be preferable to make a `Chip` wrapper in `packages/x-data-grid/src/material/index.tsx` and set the prop to `outlined` by default?

Reply:
Yes, it's better to do it that way.